src/bin/wd_copier.rs

```diff
@@ -12,10 +15,40 @@ struct Opt {
     dst: PathBuf,
 }
 
+fn get_dirty_bytes() -> u64 {
+    let meminfo = procfs::Meminfo::new().unwrap();
+    meminfo.dirty
+}
+
+fn sync_progress_bar(
```

Comment:
docstring describing what this func is doing would be good

Reply:
Is this the right way to do it? 464bf28

Reply:
This can be a little less formal -- it's not a library so we're not generating html docs. I would reword the docstring to talk more about what it's doing -- using the dirty value from `/proc/meminfo` to show a progress bar that estimates the time remaining for the sync to complete. You don't need to explicitly mention the arguments, especially `progress_bar` which is self explanatory.

Reply:
As a general rule if my function args are getting complex I repack them into a struct. For example you could have a `struct DirtyProgress` with `dirty_before_copy` and `dirty_after_copy` fields, and also have a method for getting the current estimated percent progress. I'm not sure if it's worth doing that here, but throwing it out as a possibility.

Reply:
How is this for the docstring? 0a98bc1

Reply:
Err, updated again in e5a078d since I realized that the docstring was implying the function will create the thread.